### DISCUSSION TITLE
No sound again at startup

### DISCUSSION THREAD

**Discussion by Jean-Gabriel1982** [GrandOrgue/grandorgue#1090] [General]
Hello all,

I had to completely reinstall my system and I put Linux Mint Cinnamon (instead of Xfce) as my system shut down in 2-3mn instead of 5 seconds before ..
Then my problem was not resolved but I have a clean install 😄 

I install again GO but I don't have any sound at startup. 
When I go to Setting>Audio, GO completely freeze. 
I force the closing and at re start, I have the sound ...

**Comment by oleg68** [comment]
@Jean-Gabriel1982 Please repeat the steps mentioned in https://github.com/GrandOrgue/grandorgue/discussions/855#discussioncomment-1673807

**Reply by Jean-Gabriel1982** [reply]
Yes I've just tried by 
Systemctl stop --user pasuspender.socket and service ...

My organ is gone and it seems installation of GO is new .. I have to completely reinstall my organ, keyboards, expressive pedal etc ...

**Reply by Jean-Gabriel1982** [reply]
I opened my last used organ (Caen) and after another one ..

Caen is not closed and I can play with both organs at the same time. 

I thought GO close an organ when user opened new one ..

**Reply by Jean-Gabriel1982** [reply]
Ok after re start everything (except sound) become normal

**Comment by oleg68** [comment]
@Jean-Gabriel1982 Does grandorgue hang on load? Does sound work?

**Reply by Jean-Gabriel1982** [reply]
@oleg68 No it doesn't work at startup. 
It seems it works when I close GO and restart it .. but not each time.

**Reply by oleg68** [reply]
1. run 

```
systemctl --user stop pulseaudio
```

2. Run GrandOrgue
3. Go to File->Settings->Audio
4. Post the screenshot here
5. Click to Change
6. Post the device list here

**Reply by Jean-Gabriel1982** [reply]
1. `Warning: Stopping pulseaudio.service, but it can still be activated by:
  pulseaudio.socket`

2. Running GO after closing it => sound works
3. and 4. 
![Capture d’écran du 2022-03-24 08-10-53](https://user-images.githubusercontent.com/20370038/159862196-aa90be0d-f719-4f77-bd87-59be19c47c57.png)

5. and 6. 
![Capture d’écran du 2022-03-24 08-11-23](https://user-images.githubusercontent.com/20370038/159862287-e6740a2c-5ee3-4a4c-a9a7-783ab508c18c.png)

**Reply by oleg68** [reply]
> 1. `Warning: Stopping pulseaudio.service, but it can still be activated by:   pulseaudio.socket`

Do the both:

```
systemctl --user stop pulseaudio.socket
systemctl --user stop pulseaudio.service
```

> 2. Running GO after closing it => sound works
> 3. and 4.

Are they your speackers or headphones connected with hdmi? I cann't see any analog connection among the device list.

1. Deselect 'PA' and 'Jack
2. Set 48000 instead of 44100
3. Do not use `default`. Change the audio to 'rt: alsa: hw:HD Audio Generic,<some number> and test which number works.
4. After you find the correct working device, reduce 1024 to a lower number (512, 256, 128 etc) for a lower latency.
5. After GO works with pulseaudio stopped, restart pulseaudio and make a starting script with pasuspender as was described in #855

**Reply by Jean-Gabriel1982** [reply]
@oleg68 Thank you very much Oleg for your help. 
Now I will keep the steps in my computer for next time 😄 

Everything is good now. 
Maybe the problem was the script I use at startup is the same we have used in #855 .. pasuspender was called and was not deselected in GO. 

Now, PA and Jack are deselected and I use the rt: alsa: hw:HD Audio Generic 0 and it works well !
